akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -471,6 +480,8 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
   private def startAeron(): Unit = {
     val ctx = new Aeron.Context
 
+    ctx.driverTimeoutMs(settings.Advanced.DriverTimeout.toMillis)
```

Comment:
Aha! I was thinking about this earlier, that it failed with a shorter timeout than the value in `reference.conf`...